src/main/scala/net/facetz/mailru/MailRuExport.scala

```diff
@@ -60,6 +62,20 @@ object MailRuExport extends SimpleLogger {
       } else if (ConfigHolder.config.upload) {
         new Exporter().startProcessing()
       } else {
+        possibleBadFlags = true
+      }
+    }
+
+    if (ConfigHolder.config.clean && ConfigHolder.config.expiryPeriodInDays.isDefined) {
```

Comment:
It's separate flow, use common if/else logic for this feature (like above), please